cmd/root.go

```diff
@@ -71,6 +71,7 @@ func Execute(ctx context.Context, version string) {
 	rootCmd.PersistentFlags().StringVarP(&input.githubInstance, "github-instance", "", "github.com", "GitHub instance to use. Don't use this if you are not using GitHub Enterprise Server.")
 	rootCmd.PersistentFlags().StringVarP(&input.artifactServerPath, "artifact-server-path", "", "", "Defines the path where the artifact server stores uploads and retrieves downloads from. If not specified the artifact server will not start.")
 	rootCmd.PersistentFlags().StringVarP(&input.artifactServerPort, "artifact-server-port", "", "34567", "Defines the port where the artifact server listens (will only bind to localhost).")
+	rootCmd.PersistentFlags().BoolVarP(&input.alwaysRunCheckoutAction, "always-run-checkout-action", "", false, "always run the checkout action")
```

Comment:
How about `--no-skip-checkout`? Does that make it better?

Reply:
Either `--no-skip-checkout` or `--force-remote-checkout` is fine

Reply:
Flag description should start with capitalised letter and if it could be rephrased a bit, e.g. `Checkout actions remote repository even if it's cached`

Reply:
Eventually we could introduce flag `--invalidate-actions-cache` which removes cached actions thus making act check them out

Reply:
Hm, but this flag isn't only about other actions, but also about the repository. In our case for example, we're using the full payload that GitHub sends and want act to run the `actions/checkout` to check out the ref specified in the event payload instead of using the local working directory.
So I think something like `Do not skip actions/checkout` as message would be better

Reply:
ah, I haven't actually looked into diff and from all said before I assumed it's about actions checkouts not `actions/checkout`

Reply:
Disregard the part about actions checkout then, capitalisation and flag name still applies

Reply:
I actually used `--remote-checkout` as cli flag name outside of act, with `Do not inject localcheckout into your workflows, always use the original actions/checkout` as description. _In this case I actually added an additional step before all actions/checkout steps of all jobs, without modifing the `actions/runner`._

I'm also ok with `--force-remote-checkout` or `--no-skip-checkout`.

Reply:
its `--no-skip-checkout` now! :)